### PR TITLE
Add travis config

### PR DESCRIPTION
**Pull request by seleregb**
#5 

1.  Setup travis .yml file with OSX configurations
2.  Setup `__tests__` directory with sample test file as part of travis build

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# cache directories
+cache:
+  directories:
+    - $HOME/.cache/pip3
+# clear cache log files
+before_cache:
+  - rm -f $HOME/.cache/pip3/log/debug.log
+python:
+- "3.6"
+before_install:s
```
Comment (mre) [inline .travis.yml]: was that `s` a typo?

Comment (mre) [reply .travis.yml]: I guess it should just be `before_install:`

**Comment by mre** [comment]
Fixed what I think was a typo, but the rest looked great. So I merged it. Thanks @seleregb 😊

File: .travis.yml (hunk 1 above)
Comment (seleregb) [reply .travis.yml]: lol yeah it was 😄

**Comment by seleregb** [comment]
happy to help

**Pull request merged**